api/HTMLMarqueeElement.json

```diff
@@ -149,7 +150,7 @@
           },
           "status": {
             "experimental": false,
-            "standard_track": true,
+            "standard_track": false,
```

Comment:
FYI, @Elchi3 Marquee events are also not standard track - removed from spec https://github.com/whatwg/html/pull/6343.

Otherwise looks good.

Reply:
We decided a while ago to only record event compat data once and not have the `onXXX` handlers additionally anymore.